tests/test_clnrest.py

```diff
@@ -0,0 +1,20 @@
+from ephemeral_port_reserve import reserve
+from fixtures import *  # noqa: F401,F403
+from pyln.testing.utils import wait_for
+import requests
+from pathlib import Path
+
+
+def test_clnrest_self_signed_certificates(node_factory):
+    """Test that self-signed certificates have `rest-host` IP in Subject Alternative Name."""
+    rest_port = str(reserve())
+    rest_host = '127.0.0.2'
+    base_url = 'https://' + rest_host + ':' + rest_port
```

Comment:
nit: `f'https://{rest_host}:{rest_port}'

Slowly but we are migrating tho this version here. I thin with this great organization of commits, you can fix easily with a fixup commit :) but feel free also to ignore this suggestion

Reply:
I fixed it up!